docs/maintainers/readme.md

```diff
@@ -0,0 +1,48 @@
+# Maintainer Processes
+
+This document contains basic instructions for the maintainers of the DEV
+application. It's a work in-progress, but should allow the DEV Core Team to
+document processes and strategies in a transparent way!
+
+# Skipping CI
+
+Some small PRs might seem like good candidates for skipping the CI suite. For
+example, a change that only touches documentation copy or a tiny CSS tweak.
+
+For the time being, we should avoid skipping CI in PRs because it causes some
+confusion during the review process. However, it does make sense to skip CI on
+merge commits that include extremely minor changes.
+
+You can skip CI by appending [ci skip] to merge commit's message.
+
+When in doubt, don't skip CI.
```

Comment:
I think for external contributors, since they are never merging, they should never be using the [ci skip] flag. If they do that limits who can merge their PRs. I know that will cost us some builds but it at least allows anyone on the core team then to merge the PR which I think is better for our merging process and moving PRs. What do you think?

Reply:
That's a good point I've never thought about @mstruve . Also, forcing all PRs to go through Travis will shine lights on flaky specs.